scripts/deploy/main.bicep

```diff
@@ -304,10 +308,6 @@ resource appServiceWebConfig 'Microsoft.Web/sites/config@2022-09-01' = {
         name: 'Logging:ApplicationInsights:LogLevel:Default'
         value: 'Warning'
       }
-      {
-        name: 'ApplicationInsights:ConnectionString'
-        value: appInsightsWeb.properties.ConnectionString
-      }
```

Comment:
Is this not being used?

Reply:
No.  We use the standard APPLICATIONINSIGHTS_CONNECTION_STRING instead